docs/rules/prefer-spread.md

```diff
@@ -0,0 +1,51 @@
+# Suggest using of the spread operator instead of `.apply()` callings. (prefer-spread)
+
+Up to now, we had been using `Function.prototype.apply()` callings for the variadic arguments.
+
+```js
+var args = [1, 2, 3, 4];
+Math.max.apply(Math, args);
+```
+
+Since ES2015, we can use the spread operator for the variadic arguments.
+
+```js
+var args = [1, 2, 3, 4];
+Math.max(...args);
+```
+
+## Rule Details
+
+This rule is aimed to flag `Function.prototype.apply()` callings can be replaced to the spread operator.
```

Comment:
This rule is aimed to flag usage of `Function.prototype.apply()` that can be replaced with the spread operator.